lib/utils/__tests__/addSemicolonForEditInfo.test.mjs

```diff
@@ -0,0 +1,189 @@
+import postcss from 'postcss';
+
+import addSemicolonForEditInfo from '../addSemicolonForEditInfo.mjs';
+
+describe('addSemicolonForEditInfo', () => {
+	describe('declaration', () => {
+		test('in the middle of a block having no trailing semicolon', () => {
+			const node = postcss.parse('a { color: red; top: 0px }').first.first;
+
+			const fixData = {
+				range: [0, 0],
+				text: node.toString(),
+			};
+
+			expect(fixData.text).toBe('color: red');
```

Comment:
[suggestion] I think `expect(fixData.text)` is testing `node` (PostCSS AST object), not Stylelint, so it seems unnecessary. Is simplifying the code possible like this?

```suggestion
				text: 'color: red',
			};
```

Reply:
My concern was that the PostCSS internal implementation might change.
This test verifies that `node.toString()` is lacking the `;`.

If PostCSS ever changes it's implementation this test would fail.

Reply:
Make sense, let's keep it as-is. 👍🏼 

Thank you for the feedback!